### PR TITLE
Pre-release v0.0.17

### PR DESCRIPTION
**Pull request by calvinaco**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## v0.0.17
+- Add Software Upgrade Proposal and Text Proposal transaction support
```
Comment (cdc-Hitesh) [inline CHANGELOG.md:6]:
```suggestion
- Add Software Upgrade Proposal, Text Proposal & Cancel Software Upgrade transaction support
```

**Review by cdc-Hitesh — DISMISSED** [review]
CancelSoftwareUpgradeProposal

**Review by cdc-Hitesh — COMMENTED** [review]
Shall we add a statement for cosmos-SDK v0.42.4 types support?

**Comment by calvinaco** [comment]
> cosmos-SDK v0.42.4 types support

Good point, updated here
cosmos-SDK v0.42.4 types support

**Pull request merged**